pkg/apiserver/middlewares/v1/jwt.go

```diff
@@ -59,27 +58,19 @@ func (j *JWT) authTLS(c *gin.Context) (*authInput, error) {
 	ret := authInput{}
 
 	if j.TlsAuth == nil {
-		c.JSON(http.StatusForbidden, gin.H{"message": "access forbidden"})
-		c.Abort()
+		err := errors.New("tls authentication required")
+		log.Warn(err)
 
-		return nil, errors.New("TLS auth is not configured")
+		return nil, err
 	}
 
-	validCert, extractedCN, err := j.TlsAuth.ValidateCert(c)
+	extractedCN, err := j.TlsAuth.ValidateCert(c)
 	if err != nil {
-		log.Error(err)
-		c.JSON(http.StatusForbidden, gin.H{"message": "access forbidden"})
```

Comment:
to check

Reply:
The forbidden error is raised here https://github.com/crowdsecurity/crowdsec/blob/crl/pkg/apiserver/middlewares/v1/api_key.go#L145

and tested here https://github.com/crowdsecurity/crowdsec/blob/crl/test/bats/11_bouncers_tls.bats#L196

As the comment says, since we could not validate the identity I think Unauthorized is more appropriate, but it's splitting hair.